migration/identity-migration/src/main/java/io/camunda/migration/identity/config/saas/StaticEntities.java

```diff
@@ -152,4 +158,113 @@ public class StaticEntities {
               Set.of(PermissionType.READ))
           // TODO: add document permissions once implemented
           );
+
+  public static final List<CreateAuthorizationRequest> CLIENT_PERMISSIONS =
+      List.of(
+          // ZEEBE
+          new CreateAuthorizationRequest(
+              ZEEBE_CLIENT_ID,
+              AuthorizationOwnerType.CLIENT,
+              "*",
+              AuthorizationResourceType.MESSAGE,
+              Set.of(PermissionType.CREATE)),
+          new CreateAuthorizationRequest(
+              ZEEBE_CLIENT_ID,
+              AuthorizationOwnerType.CLIENT,
+              "*",
+              AuthorizationResourceType.SYSTEM,
+              AuthorizationResourceType.SYSTEM.getSupportedPermissionTypes()),
+          new CreateAuthorizationRequest(
+              ZEEBE_CLIENT_ID,
+              AuthorizationOwnerType.CLIENT,
+              "*",
+              AuthorizationResourceType.RESOURCE,
+              Set.of(
+                  PermissionType.CREATE,
+                  PermissionType.DELETE_FORM,
+                  PermissionType.DELETE_PROCESS,
+                  PermissionType.DELETE_DRD,
+                  PermissionType.DELETE_RESOURCE)),
+          new CreateAuthorizationRequest(
+              ZEEBE_CLIENT_ID,
+              AuthorizationOwnerType.CLIENT,
+              "*",
+              AuthorizationResourceType.PROCESS_DEFINITION,
+              Set.of(
+                  PermissionType.UPDATE_PROCESS_INSTANCE,
+                  PermissionType.UPDATE_USER_TASK,
+                  PermissionType.CREATE_PROCESS_INSTANCE,
+                  PermissionType.DELETE_PROCESS_INSTANCE)),
+          new CreateAuthorizationRequest(
+              ZEEBE_CLIENT_ID,
+              AuthorizationOwnerType.CLIENT,
+              "*",
+              AuthorizationResourceType.DECISION_DEFINITION,
+              Set.of(
+                  PermissionType.CREATE_DECISION_INSTANCE,
+                  PermissionType.DELETE_DECISION_INSTANCE)),
+          new CreateAuthorizationRequest(
+              ZEEBE_CLIENT_ID,
+              AuthorizationOwnerType.CLIENT,
+              "*",
+              AuthorizationResourceType.DECISION_REQUIREMENTS_DEFINITION,
+              Set.of(PermissionType.UPDATE, PermissionType.DELETE)),
+          // OPERATE
+          new CreateAuthorizationRequest(
+              OPERATE_CLIENT_ID,
+              AuthorizationOwnerType.CLIENT,
+              "*",
+              AuthorizationResourceType.MESSAGE,
+              Set.of(PermissionType.READ)),
+          new CreateAuthorizationRequest(
+              OPERATE_CLIENT_ID,
+              AuthorizationOwnerType.CLIENT,
+              "*",
+              AuthorizationResourceType.BATCH_OPERATION,
+              Set.of(PermissionType.READ, PermissionType.CREATE, PermissionType.UPDATE)),
```

Comment:
💭 delete does not exist but is defined in https://docs.google.com/document/d/1ikxw9ARCv-MBwASnj7WhW5uHCndCRk2B35xi-cB2N0M/edit?tab=t.0#heading=h.vyle3unzbdnq should we raise that to the team to clarify if it was missed or the doc needs to change?